### PR TITLE
Branding

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request

Adds branding. Now in the HOS locker.

## Proof Of Testing

https://github.com/Bubberstation/Bubberstation/assets/77420409/34595b0f-46a7-41b3-b4e0-7dc897b41931

## Changelog

:cl:
add: HOS gets a sizzling branding knife, for the worst criminals NT has to offer.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/branding/brand.dm [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	if(!istype(victim, /mob/living/carbon))
+		return
+	var/datum/status_effect/branded/current_branding = tgui_input_list(attacker, "What crime are you branding them with?", "Branding", branding_with) // todo, make names show
+
+	if(!attacker.combat_mode && do_after(attacker, 6 SECONDS, victim))
+		victim.apply_status_effect(current_branding)
+		to_chat(attacker, span_userdanger("You brand [victim] with the mark of [initial(current_branding.id)]"))
+
+		var/obj/item/bodypart/branded_limb = victim.get_bodypart(deprecise_zone(BODY_ZONE_CHEST))
+		branded_limb.force_wound_upwards(/datum/wound/burn/flesh/critical, wound_source = "Branding")
```
Comment (nikothedude) [inline modular_zubbers/code/modules/branding/brand.dm:28]:
```
victim.cause_wound_of_type_and_severity(WOUND_BURN, branded_limb, WOUND_SEVERITY_CRITICAL))
```

same effect except it supports synthetic wounds

File: modular_zubbers/code/modules/branding/brand.dm [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+/* 	var/list/preset_brandable_crimes = list()
+	for(var/i in branding_with)
+		preset_brandable_crimes |= i.initial(id) */
+	if(!istype(victim, /mob/living/carbon))
+		return
+	var/datum/status_effect/branded/current_branding = tgui_input_list(attacker, "What crime are you branding them with?", "Branding", branding_with) // todo, make names show
+
+	if(!attacker.combat_mode && do_after(attacker, 6 SECONDS, victim))
+		victim.apply_status_effect(current_branding)
+		to_chat(attacker, span_userdanger("You brand [victim] with the mark of [initial(current_branding.id)]"))
```
Comment (nikothedude) [inline modular_zubbers/code/modules/branding/brand.dm:25]: missing a period

File: modular_zubbers/code/modules/branding/brand.dm (hunk 1 above)
Comment (nikothedude) [reply modular_zubbers/code/modules/branding/brand.dm:28]: you should use this anyway, this is the defacto "give a limb a wound" proc

source: i refactored wounds and added t his proc

**Comment by nikothedude** [comment]
also, what happened to the feature freeze? isnt this a feature?

**Comment by nikothedude** [comment]
theres also no justification for why this is good for the game

**Comment by YakumoChen** [comment]
> also, what happened to the feature freeze? isnt this a feature?

We aren't in a freeze

**Comment by Kepteyn** [comment]
While not _inherently_ terrible, it seems to be a thing that exists to make antag's lives harder for the sake of making it harder. Hard to try to escape from perma when anyone who sees you can see that you're a prisoner, even if you changed your clothes and got a fake ID. If it can be removed - which, being as it is a wound, it may - that isn't quite as bad, but its still another thing that just exists for the sake of making the lives of antags harder.

My biggest concern would be lore-wise: _**WHY**_ does the HOS and security have access to a cattlebrand? Is it universal procedure amongst the GalFed to brand prisoners for their crimes? That seems a particularly barbaric practice if so, and I'd hope some justification for why that's the case would exist. Does GalFed not care how corporations handle security affairs in the frontier? I would expect security to be far less restricted in its use of violence if so. As it stands, the brand just seems like a random, out-of-place tool with little justification for _**why**_ it exists.

Being able to permanently brand someone for live as a criminal because 'he disrespected me' is pretty based though, gotta give that much.

**Comment by Cyprex** [comment]
This might not make you uncomfortable, but it makes me very uncomfortable. It's not a good idea to give sec ideas to torture prisoners(who are generally mechanically largely unable to fight back). This will absolutely end up used in a way that will make people uncomfortable. It's a thing that is largely there for RP reasons, it's excessively cruel and so on and so forth. Tracking implants are a better option mechanically- this is just cruel. To use this item is to consciously decide to torture another player's character. Some might be fine with this. Others not so much, and by existing this item encourages this behavior and essentially communicates to security players that this is a fine thing to do.

**Comment by Cyprex** [comment]
A stamp or similiar non-torturous mark would be fine, feel free to reopen this with that

**Pull request closed without merging**